drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/columns/control/ColumnManagementView.java

```diff
@@ -36,15 +39,24 @@
 import org.drools.workbench.screens.guided.dtable.client.resources.i18n.GuidedDecisionTableConstants;
 import org.drools.workbench.screens.guided.dtable.client.widget.table.GuidedDecisionTableModellerView;
 import org.drools.workbench.screens.guided.dtable.client.widget.table.utilities.ColumnUtilities;
-import org.uberfire.ext.widgets.common.client.common.popups.YesNoCancelPopup;
 import org.uberfire.ext.widgets.common.client.common.popups.errors.ErrorPopup;
-import org.uberfire.mvp.ParameterizedCommand;
 
+@Dependent
 public class ColumnManagementView extends VerticalPanel {
 
+    private Instance<DeleteColumnManagementAnchorWidget> deleteColumnManagementAnchorWidgets;
```

Comment:
Sorry.. I missed this last time around.. best to make this ```ManagedInstance``` to ensure instances are released.